producer/src/main/kotlin/streams/StreamsExtensionFactory.kt

```diff
@@ -67,6 +70,9 @@ class StreamsEventRouterLifecycle(private val db: GraphDatabaseAPI,
 
     private fun registerTransactionEventHandler() {
         if (streamsEventRouterConfiguration.enabled) {
+            if (!KafkaValidationUtils.isAutoCreateTopicsEnabled(KafkaConfiguration.from(configuration.config).asProperties())) {
```

Comment:
per favore salva `KafkaConfiguration.from(configuration.config).asProperties()` in una variabile